c++/src/linked-list.cc

```diff
@@ -254,6 +263,11 @@ int main() {
   list.printTo(listString);
   assert(listString.str() == "0 → 1 → 2 → 3 → 4 → 5 → 6"); listString.str("");
 
+  LinkedList<int> initializer_list_list = LinkedList<int>{0, 1, 2, 3};
```

Comment:
An `initializer_list` constructor lets you initialize things much simpler than this. This should be either:

``` cpp
LinkedList<int> list = {0, 1, 2, 3};
```

Or even better and much preferred:

``` cpp
LinkedList<int> list{0, 1, 2, 3};
```

Notice that it's convention to not use a space between the `list` and the `{`. Treat it similar to if you were calling a regular constructor e.g. `LinkedList<int> someList();`.